test/test_bulk_renewal.js

```diff
@@ -0,0 +1,106 @@
+const ENS = artifacts.require('@ensdomains/ens/ENSRegistry');
+const HashRegistrar = artifacts.require('@ensdomains/ens/HashRegistrar');
+const PublicResolver = artifacts.require('@ensdomains/resolver/PublicResolver');
+const BaseRegistrar = artifacts.require('./BaseRegistrarImplementation');
+const ETHRegistrarController = artifacts.require('./ETHRegistrarController');
+const SimplePriceOracle = artifacts.require('./SimplePriceOracle');
+const BulkRenewal = artifacts.require('./BulkRenewal');
+var Promise = require('bluebird');
+
+const namehash = require('eth-ens-namehash');
+const sha3 = require('web3-utils').sha3;
+const toBN = require('web3-utils').toBN;
+
+const NULL_ADDRESS = "0x0000000000000000000000000000000000000000"
+const ETH_LABEL = sha3('eth');
+const ETH_NAMEHASH = namehash.hash('eth');
+
+const advanceTime = Promise.promisify(function(delay, done) {
```

Comment:
we have a library for this type of stuff. https://github.com/ensdomains/test-utils